kas/projects/tedge-rauc.yaml

```diff
@@ -50,14 +50,9 @@ repos:
 
   meta-rauc-community:
     url: "https://github.com/thin-edge/meta-rauc-community"
-    branch: kirkstone-fork
     layers:
       meta-rauc-raspberrypi:
```

Comment:
The usage of the fork is still required. It was mainly done to provide a custom WIC file to change which partition is marked for growing (expanding on first boot). The meta-rauc-community sets the `/home/root` partition to be expanded, where as I prefer to expand the `/data` partition. Though if we can find a better way to provide a new WIC definition file that does not involve forking the repo, then by all means.

Reply:
In that case we need to create PR under the `meta-rauc-community` to add `scarthgap` as compatible layer (or create separate fork for scarthgap).